std/variant.d

```diff
@@ -1424,22 +1423,10 @@ unittest
     assert(b[1] == 3.14);
 }
 
-/** Code that needs functionality similar to the $(D boxArray)
-function in the $(D std.boxer) module can achieve it like this:
-*/
-unittest
```

Comment:
`std.boxer` doesn't (seem to) exist, for some time. The example code also doesn't actually provide anything specific over what variadic template functions provide.